opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1481,32 +1497,41 @@ namespace Opm
         EvalWell control_eq(0.0);
 
         switch (well_controls_get_current_type(well_controls_)) {
-            case THP: // not handling this one for now
-            {
-                OPM_DEFLOG_THROW(std::runtime_error, "Not handling THP control for Multisegment wells for now", deferred_logger);
-            }
-            case BHP:
-            {
-                const double target_bhp = well_controls_get_current_target(well_controls_);
-                control_eq = getSegmentPressure(0) - target_bhp;
-                break;
+        case THP: // not handling this one for now
```

Comment:
Really sorry for the nittpicking. It is not the purpose of the original comment.  You can create a different pull request if you want to address the indentation of the switch in the whole file, that is perfectly fine. 

It does not makes the pull request easier to read by changing all the indentations here. It is not the style of the switch for this file either.

Reply:
>not the style of the switch for this file either.

The file is not consistent with indentation for switches. I choose one of the styles used. I don't have any plan to make a PR to clean-up the well model for indentations. I tried to fix it the way you wanted it to be. I suggest that you make a PR later to synchronize the style in the file, then you get it the way you want.

Reply:
Is it the way you want know?

Reply:
It is better than the previous version.  The code is pretty complicated now. If you would like to, you can put the surface rate control in one function, even split them to two functions for injectors and producers. Then the code will be easier to read.  You do not have to do it. Just let me know.